templates/.jshintrc

```diff
@@ -8,17 +8,10 @@
     "access": false,
     "channel": false,
     "customActionStub": false,
-    "getEffectiveOldDoc": false,
```

Comment:
These utility functions are now passed in an object parameter to the sync function template's submodule functions so there is no need to define them as global variables anymore.